UnityProject/Assets/Scripts/Managers/NetworkManagement/CustomNetworkManager.cs

```diff
@@ -176,12 +176,20 @@ public void AuthChange(ulong steamid, ulong ownerid, ServerAuth.Status status)
 			return;
 		}
 
+		//god please work, should be the leave message and associated logging.
+		if (status == ServerAuth.Status.VACCheckTimedOut)
+		{
+
+			Logger.LogWarning($"The SteamID '{SteamID}' left the server. ({status})");
+			return;
+	}
+
 		if (status == ServerAuth.Status.VACCheckTimedOut)
 		{
 			return;
 		}
 
-		Kick(player, $"Steam: {status}");
+		Kick(player, $"{player} has departed from the station.");
```

Comment:
I don't think we should be broadcasting over chat if someone has left. Chat is meant to resemble radio chatter or local room chatter, saying that someone has left or logged out will break immersions unless it's broadcast over OOC but I still think we should not notify others of joining or leaving. Just remove the whole kick line, it's not needed.

Reply:
Done.